lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -762,6 +762,9 @@ defmodule ExUnit.Assertions do
   Asserts `expression` will exit.
   Returns the exit status/message or fails otherwise.
```

Comment:
What do tou think about updating this line with “Returns the exit status/message of the current process or fails otherwise” and remove the next line?